Samples/3b-custom-transformations/generated/pipeline.yaml

```diff
@@ -0,0 +1,88 @@
+---
+pipeline-emitter:
```

Comment:
pipeline representation is fully serializable, think we can create a nice diagnostic "plugin" that'll visualize this DAG and show live information about it!